crates/tabby-common/src/config.rs

```diff
@@ -145,6 +146,12 @@ impl Default for ServerConfig {
 #[async_trait]
 pub trait RepositoryAccess: Send + Sync {
     async fn list_repositories(&self) -> Result<Vec<RepositoryConfig>>;
+
+    fn start_snapshot(&self) -> u64 {
+        Default::default()
+    }
```

Comment:
```suggestion
    fn start_snapshot(&self， version: u64){
    }
```

Reply:
Starting the snapshot is a no-op, it just generates the next version key

Reply:
let's generate the new version in scheduler/dataset (with current timestamp). Leaving it as no-op is fine. mostly for a provider (maybe just for logging)